Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -103,6 +104,14 @@ export default class LinearTimelineComponent extends Vue {
             self.setPageFromDate(eventDate);
         });
 
+        this.eventBus.$on(
+            EventMessageName.TimelineCurrentDateUpdated,
+            function (currentDate: Date) {
+                self.setPageFromDate(currentDate);
```

Comment:
this will probably not work since the date is not guaranteed to exist on that month.

Reply:
More specifically below will give errors
```
let self = this;
        // Wait for next render cycle until the pages have been calculated and displayed
        this.$nextTick().then(function () {
            const date = new Date(eventDate).setHours(0, 0, 0, 0);
            let container: HTMLElement[] = self.$refs[date] as HTMLElement[];
            container[0].focus();
        });
```

Reply:
I have removed these temp codes which need to be cleaned up as a part of the 8693